lib/galaxy/webapps/galaxy/api/configuration.py

```diff
@@ -85,7 +85,7 @@ def tool_lineages(self, trans):
 
     @expose_api
     @require_admin
-    def reload_toolbox(self, trans):
+    def reload_toolbox(self, trans, **kwds):
```

Comment:
I introduced this change to be able to call this from the API -- otherwise it would pass the 'key' param containing the API key to `reload_toolbox`, which would just fail.